### DISCUSSION TITLE
Listing my uploaded images

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#21] [Q&A]
I'm trying out the image endpoint with the API.  I can upload an image and insert images in the web editor. Is there any way to get a list of my images? I suspect the current API is dumping everything into a giant bucket. It would be helpful if I could "browse" my images as I occasionally repeat them in a newsletter, and the efficient part of me hates the idea of uploading another copy of an existing file.

**Comment by jmduke** [comment]
Not at the moment, but definitely a reasonable request if you're just looking to iterate over them via the API! Want me to add a LIST /v1/images endpoint?

**Comment by jdhitsolutions** [comment]
I think it makes sense. For example, if I accidentally upload the wrong image, I'd like to be able to remove it. That's just one use case. Of course, if you open this can of worms, the next logical request will be an image browser in the web interface. 😁

**Reply by jmduke** [reply]
Ha, that part's unlikely ;)

**Comment by jmduke** [comment]
Filed here.

https://github.com/buttondown-email/roadmap/issues/2435
